packages/@lwc/template-compiler/src/__tests__/fixtures.spec.ts

```diff
@@ -20,13 +20,14 @@ describe('fixtures', () => {
         },
         ({ src, dirname }) => {
             const configPath = path.resolve(dirname, 'config.json');
+            const filename = path.basename(dirname);
 
-            let config: Config = {};
+            let config: Config = { namespace: 'x', name: filename };
             if (fs.existsSync(configPath)) {
-                config = require(configPath);
+                config = { ...config, ...require(configPath) };
             }
 
-            const compiled = compiler(src, config);
+            const compiled = compiler(src, filename, config);
```

Comment:
The new snapshots all have the implicit imports, which can be noisey. 

As an alternative we could add something to cut out the imports for certain snapshots.

I opted to leave it as-is for now to get opinions.

Reply:
I'm fine with the snapshots being a bit noisy. I actually prefer to see what the code actually looks like, rather than trying to hide boilerplate/repetition.